workflow-core/src/commonMain/kotlin/com/squareup/workflow1/StatefulWorkflow.kt

```diff
@@ -76,6 +76,16 @@ public abstract class StatefulWorkflow<
   public inner class RenderContext internal constructor(
     baseContext: BaseRenderContext<PropsT, StateT, OutputT>
   ) : BaseRenderContext<@UnsafeVariance PropsT, StateT, @UnsafeVariance OutputT> by baseContext {
+    public inline fun <reified CurrentStateT : StateT & Any> safeEventHandler(
```

Comment:
not sure I understand why this one isn't deprecated? (Or why its introduced?)

Reply:
Oops, lost the deprecation annotation. Thanks.